src/Learners/Core/Utilities.cs

```diff
@@ -28,7 +26,8 @@ public static class Utilities
         /// </summary>
         /// <param name="learner">The learner to serialize.</param>
         /// <param name="fileName">The name of the file.</param>
-        public static void Save(this ILearner learner, string fileName)
+        /// <param name="formatter">The formatter to use.</param>
+        public static void Save(this ILearner learner, string fileName, IFormatter formatter)
```

Comment:
> unnecessarily

What's the alternative? There is no formatter (apart from BinaryFormatter) that serializes all learners. This allows people to write BinaryFormatter if they like (which can't be here because of compliance) but also providing a formatter that works for their specific learner. I don't see an alternative other than writing and maintaining something like BinaryFormatter; or rewriting all of our learners to work with Newtonsoft.Json or similar, which is a big change that doesn't seem necessary.

Reply:
So this is only here to support BinaryFormatter?  Then it should go.